qHttpServerBin/5.15.2/ios/include/QtHttpServer/qhttpserver.h

```diff
@@ -0,0 +1 @@
+#include "../../../qthttpserver/src/httpserver/qhttpserver.h"
```

Comment:
@p3g4asus i was trying to add qhttpserver on iOS (i didn't yet since this PR). But the compiler fails when builds this file (qhttpserver.h). Infact it points to a "../../../qthttpserver/src/httpserver/qhttpserver.h" that is not commited. Probably from the qhttpserver GIT repository? So i need it too? We should add it as a subclone so, what do you think?

Reply:
Sorry:  I forgot to one installation step:
Copy cloned git repository (with the added 3rdparty/http-parser files) inside $QTDIR/$QTVERSION/Src. NB the parent folder name of the cloned git repository MUST BE qthttpserver.
Sorry for the mistake.

Reply:
I edited my original message fixing the links. They renamed the default branch from master to main.